.circleci/config.yml

```diff
@@ -958,8 +958,19 @@ jobs:
             terraform destroy -auto-approve
       - run: *notify-slack-failure
 
+  noop:
```

Comment:
Maybe comment that this is used to 'probe' if the tests are actually run? This and the verify-ci workflow below may become separated in the file.

Reply:
I wish it did probe. Unfortunately it really is just a no-op job because a workflow needs at least one job.